src/Http/Client/Response.php

```diff
@@ -411,7 +411,7 @@ public function getJson()
     /**
      * Get the response body as JSON decoded data.
      *
-     * @return mixed
+     * @return array<mixed>
```

Comment:
This is incorrect since JSON decoded data doesn't always have to be an array. Return type of `json_decode()` is `mixed`.

Reply:
In theory yes, but not in practical terms here, the response is always expected later to be an assoc array
But I reverted it for now.
In the future we might want to throw an exception for unexpected types maybe.